schemas/openshift/terraform-resource-1.yml

```diff
@@ -139,6 +139,13 @@ properties:
       type: string
   image:
     type: object
+    properties:
+      upstream:
+        type: object
+        properties:
```

Comment:
isn't `name` missing up here? i'm unsure if it is really necessary though. i was not able to find respective openapi docs. up until now i only dealt with examples where the config in the top level `properties` section matches the ones in the `oneOf` section